### PR TITLE
Add version number to settings page also

### LINKED ISSUE #130: Show app version info in settings
Would be awesome to prefill that in bug reports, but not sure if that's possible.

### PR DESCRIPTION
**Pull request by Sherlouk**
Closes #130 

Includes some minor tweaks to cell spacing to make it look nice, and a new LabelCell

Moved the version string stuff into it's own extension so it could be reused (and added localization to it because Version in Spanish is Versión and they are totally different words!)

Looks like this:
Ooh fancy

### CODE REVIEW

File: Classes/Views/LabelCell.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+final class LabelCell: UICollectionViewCell {
+    
+    let label = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        contentView.backgroundColor = .clear
+        accessibilityTraits |= UIAccessibilityTraitStaticText
```
Comment (BasThomas) [inline Classes/Views/LabelCell.swift]: 😍

Comment (Sherlouk) [inline Classes/Views/LabelCell.swift]: Admittedly wouldn't have thought to do it if it wasn't for the fact I copied it from the button cell 😅 😅

Comment (BasThomas) [inline Classes/Views/LabelCell.swift]: That works too, right? :)

File: Classes/Utility/Bundle+Version.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    
+    var buildNumber: String? {
+        return infoDictionary?["CFBundleVersion"] as? String
+    }
+    
+    var prettyVersionString: String {
+        return NSLocalizedString("Version", comment: "") + " \(versionNumber ?? "Unknown") (\(buildNumber ?? "0"))"
+    }
+    
+}
```
Comment (rnystrom) [inline Classes/Utility/Bundle+Version.swift]: Lovely :)

File: Classes/Utility/Bundle+Version.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    var versionNumber: String? {
+        return infoDictionary?["CFBundleShortVersionString"] as? String
+    }
+    
+    var buildNumber: String? {
+        return infoDictionary?["CFBundleVersion"] as? String
+    }
+    
+    var prettyVersionString: String {
+        return NSLocalizedString("Version", comment: "") + " \(versionNumber ?? "Unknown") (\(buildNumber ?? "0"))"
```
Comment (rnystrom) [inline Classes/Utility/Bundle+Version.swift]: Let's use `Strings.unknown` so its using common strings + localized

File: Classes/Views/LabelCell.swift [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+//  Created by Sherlock, James on 15/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class LabelCell: UICollectionViewCell {
+    
+    let label = UILabel()
```
Comment (rnystrom) [inline Classes/Views/LabelCell.swift]: `private`

File: Classes/Views/Styles.swift [hunk 5]
```diff
@@ -19,7 +19,7 @@ enum Styles {
         static let rowSpacing: CGFloat = 8
         static let cellSpacing: CGFloat = 15
         static let tableCellHeight: CGFloat = 44
-        static let tableSectionSpacing: CGFloat = 35
+        static let tableSectionSpacing: CGFloat = 17
```
Comment (rnystrom) [inline Classes/Views/Styles.swift]: This also effects the "no notifications" (:tada:) view. Double check that doesn't look too tight?

File: Classes/Settings/SettingsVersionInfoSectionController.swift [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+//
+
+import UIKit
+import IGListKit
+
+final class SettingsVersionInfoSectionController: ListSectionController {
+    
+    override init() {
+        super.init()
+        inset = UIEdgeInsets(top: 0, left: 0, bottom: Styles.Sizes.tableSectionSpacing, right: 0)
```
Comment (rnystrom) [inline Classes/Settings/SettingsVersionInfoSectionController.swift]: All the others have the inset at the `top`, why bottom here?

File: Classes/Settings/SettingsVersionInfoSectionController.swift [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    
+    override func cellForItem(at index: Int) -> UICollectionViewCell {
+        guard let cell = collectionContext?.dequeueReusableCell(of: LabelCell.self, for: self, at: index) as? LabelCell
+            else { fatalError("Collection context must be set or cell incorrect type") }
+        cell.label.text = Bundle.main.prettyVersionString
+        return cell
+    }
+    
+    override func didSelectItem(at index: Int) {}
```
Comment (rnystrom) [inline Classes/Settings/SettingsVersionInfoSectionController.swift]: can remove

File: Classes/Settings/SettingsVersionInfoSectionController.swift (hunk 6 above)
Comment (Sherlouk) [inline Classes/Settings/SettingsVersionInfoSectionController.swift]: Bottom is unnecessary here, the lack of top is to make it closer to the log out button (hence the changes in the first place)

With the original spacing it was too far away which looked weird

File: Classes/Utility/Bundle+Version.swift (hunk 3 above)
Comment (Sherlouk) [inline Classes/Utility/Bundle+Version.swift]: Done!

File: Classes/Views/LabelCell.swift (hunk 4 above)
Comment (Sherlouk) [inline Classes/Views/LabelCell.swift]: Can't have this as private as need access to it to set it's text value

File: Classes/Views/Styles.swift (hunk 5 above)
Comment (Sherlouk) [inline Classes/Views/Styles.swift]: Reverted back to old value, just changed the bottom value on Log Out to give the same effect

**Pull request merged**